index.bs

```diff
@@ -844,6 +846,8 @@ To <dfn>obtain a report</dfn> given an [=attribution source=] |source| and an [=
     :: |trigger|'s [=attribution trigger/trigger time=].
     : [=attribution report/source identifier=]
     :: |source|'s [=attribution source/source identifier=].
+    : [=attribution report/report id=]
+    :: A random string.
```

Comment:
Do we want to enforce this is a random number encoded as a string or something?  E.g. like a random 64 bit int.

Reply:
I wasn't sure if that was overly restrictive, but that would be fine with me. CC @csharrison.

Reply:
I was thinking to align with eventid, but I agree it's not necessary. E.g. a UUID seems like a good fit here.

We may want to make some statement on uniqueness, e.g.:

A new unique random string.

Reply:
Are we sure we want it to be unique? The client would have to scan the report cache to ensure that. And then we have the question of whether we need to retain a cache of already-used IDs for all time, or if it only needs to be unique among reports that haven't been sent yet.

Reply:
Good point. I guess functionally what we want is:

"A random string with high enough entropy that they should be effectively unique"

Maybe that is enough of an argument to just use a UUID?

Reply:
Changed to UUID.